src/loaders/slides-json-loader.js

```diff
@@ -1,9 +1,21 @@
 module.exports = function (content) {
   const json = JSON.parse(content);
-  const imports = json
-    .map((f, i) => `import f${i} from "./${f}?chapterIndex=${i}";`)
-    .join("\n");
-  return `${imports}\nexport default [${json
-    .map((f, i) => `f${i}`)
-    .join(", ")}]`.trim();
+  const imports = json.flatMap((filename, index) => [
+    `import content${index} from "./${filename}?chapterIndex=${index}";`,
+    `import title${index} from "./${filename}?titleOnly";`,
+  ]);
+  const chapters = json
+    .map(
+      (filename, index) =>
+        `{ index: ${index}, title: title${index}, content: content${index} }`
+    )
+    .join(",");
+  const fullContent =
+    json.map((filename, index) => `content${index}`).join(" + ") || '""';
+  const exports = [
+    `export const chapters = [${chapters}];`,
+    `export const title = chapters[0]?.title || "";`,
```

Comment:
```suggestion
    `export const title = chapters[0]?.title || "TITLE NOT FOUND";`,
```

Would be nice to keep the error message `"The title of the training was not found. Please ensure that the slides include a first level title (e.g. '# My title' or '<h1>My title</h1>') in the first file referenced in 'Slides/slides.json'."` (now in the logs of sensei)

Reply:
I'm gonna handle this in the revealjs loader.